src/main/java/org/cryptomator/frontend/fuse/locks/PathLockBuilder.java

```diff
@@ -4,6 +4,10 @@
 
 	PathLock forReading();
 
+	PathLock tryForReading() throws AlreadyLockedException;
+
 	PathLock forWriting();
 
+	PathLock tryForWriting() throws AlreadyLockedException;
+
```

Comment:
This is a significant change that should be covered by tests!

Reply:
Done by ce4c76a404eab879af6468d982328630a92fb711